packages/web-runtime/src/services/uppyService.ts

```diff
@@ -276,6 +276,16 @@ export class UppyService {
     return this.uppy.cancelAll()
   }
 
+  getCurrentUploads(): Record<string, unknown> {
+    return this.uppy.getState().currentUploads
+  }
+
+  isRemoteUploadInProgress(): boolean {
+    // only one remote upload at a time means it's sufficient to check the first file
```

Comment:
What if a local upload was started before?

Reply:
There is no scenario where we have a local and a remote upload running at the same time. However, when a file of an import failed before, that could be an issue here indeed. Let me change the condition a bit.